platform/ios/src/MGLLocationManager.m

```diff
@@ -46,6 +46,11 @@ - (void)stopUpdatingLocation {
             [self.delegate locationManagerDidStopLocationUpdates:self];
         }
     }
+    if(self.standardLocationManager.monitoredRegions.count > 0) {
+        for(CLRegion *region in self.standardLocationManager.monitoredRegions) {
```

Comment:
Since [monitoredRegions are a shared resource](https://developer.apple.com/reference/corelocation/cllocationmanager/1423790-monitoredregions) we should only call `stopMonitoringForRegion:` on the (at most) one region this class creates for itself.